src/MICore/Transports/ITransport.cs

```diff
@@ -16,6 +16,12 @@ public interface ITransport
         void Send(string cmd);
         void Close();
         bool IsClosed { get; }
+
+        /// <summary>
+        /// It is used to know whether to fake a response from the debugger
```

Comment:
How about -- 
    Process id of the debugger (gdb/lldb-mi/clrdbg) process. It is used to know...